packages/loader/container-loader/src/test/retryUtils.spec.ts

```diff
@@ -8,13 +8,24 @@ import { DriverErrorType } from "@fluidframework/driver-definitions";
 import { TelemetryNullLogger } from "@fluidframework/common-utils";
 import { runWithRetry } from "../utils";
 
+const _setTimeout = global.setTimeout;
+const fastSetTimeout: any =
+    (callback: (...cbArgs: any[]) => void, ms: number, ...args: any[]) => _setTimeout(callback, ms / 1000.0, ...args);
+
 describe("Retry Util Tests", () => {
     const deltaManager = {
         refreshDelayInfo: () => {},
         emitDelayInfo: () => {},
     };
     const logger = new TelemetryNullLogger();
 
+    before(()=>{
```

Comment:
I'd rather use workflow similar to mockFetchOk / mockFetchMultiple, i.e. replace it for duration of the wrapped callback.
That way it's a bit more cleaner in terms of scope, and we can (and likely should) do more validation, like how many calls are actually changed (see assert(cb !== undefined, "the end") & assert(responses.length === 0, "all responses used") asserts)